src/services/specification.service.ts

```diff
@@ -185,7 +184,7 @@ export class SpecificationService {
     );
   }
 
-  private static isNotSupportedVersion(rawSpec: string): boolean {
+  static isNotSupportedVersion(rawSpec: string): boolean {
```

Comment:
Is removing the `private` fixing the tests? From what I can understand it might not be? Any reason why that's changed? Unless I'm missing something 😅

Reply:
That lines fix tests https://github.com/asyncapi/studio/pull/271/files#diff-30ced4fa50892c2cded728e0ed12cfdf2b0581861d73696ae02a56cdaf80c9a6R64-R68 :) I also checked that in one test we use `isNotSupportedVersion` function and TS complains that it's private so I changed it to public (only for tests), but it didn't cause any problems at all.